AsyncDisplayKit/ASTextNode.h

```diff
@@ -270,4 +270,22 @@ typedef NS_ENUM(NSUInteger, ASTextNodeHighlightStyle) {
 
 @end
 
+/**
+ * @abstract Text node deprecated properties
+ */
+@interface ASTextNode (Deprecated)
+
+/**
+ @see attributedText
```

Comment:
Add to this and say these properties are identical / equivalent, but that attributedText is now the standard API name in order to match UILabel and ASEditableTextNode.